solana/ics27/README.md

```diff
@@ -0,0 +1,55 @@
+# Program Implementing Cross-Chain Calls Based on ICS27 Specification
+
+This document outlines the operation of a program designed to facilitate cross-chain calls in adherence to the ICS27 specification. The program enables the execution of cross-chain transactions, leveraging the Inter-Blockchain Communication (IBC) protocol for secure and verified interactions between different blockchain networks.
+
+## Workflow:
+
+### 1. Relayer Interaction:
+- A relayer, an off-chain entity, invokes the `deliver` function on the program, passing in packet data for processing.
+
+### 2. Packet Verification:
+- The program verifies that the packet has been authenticated by the IBC core program's account, ensuring its legitimacy.
+
+### 3. Packet Parsing and Account Verification:
+- It parses the packet according to the `svm/borsh-1` standard, ensuring the packet data is correctly serialized/deserialized.
```

Comment:
What’s svm/borsh-1?

Reply:
will update do. but this is 
```
ICS-27 takes advantage of [ICS-04 channel version negotiation](https://github.com/cosmos/ibc/blob/main/spec/core/ics-004-channel-and-packet-semantics/README.md#versioning) to negotiate metadata and channel parameters during the channel handshake. The metadata will contain the encoding format along with the transaction type so that the counterparties can agree on the structure and encoding of the interchain transactions.
```
So encoding is borsh, transactions type is `svm` (it can be neon-evm on solana for example) and `1` is version in case we need to bump it.

See ics20 version example.